co/objectMap.cpp

```diff
@@ -166,7 +166,6 @@ void ObjectMap::_commitMasters( const uint32_t incarnation )
 
 void ObjectMap::serialize( DataOStream& os, const uint64_t dirtyBits )
 {
-    Serializable::serialize( os, dirtyBits );
```

Comment:
The implemenation was empty (see above) and now is pure virtual. Calling it will give you an unresolved symbol.